easybuild/easyblocks/generic/rubygem.py

```diff
@@ -81,4 +81,4 @@ def test_step(self):
 
     def install_step(self):
         """Install Ruby Gems using gem package manager"""
-        run_cmd('gem install --local %s' % self.ext_src)
+        run_cmd("gem install --install-dir %s --local %s" % (self.installdir, self.ext_src))
```

Comment:
no, wait, this ain't right...

`-i, -​-install-dir DIR - Gem repository directory to get installed gems`

so, `--install-dir` is supposed to point to `$EBROOTRUBY`?! :-/